common/main/java/com/couchbase/lite/MutableArray.java

```diff
@@ -82,11 +86,26 @@ public MutableArray(List<Object> data) {
     public MutableArray setData(List<Object> data) {
         synchronized (lock) {
             internalArray.clear();
-            for (Object obj : data) { internalArray.append(Fleece.toCBLObject(obj)); }
+            for (Object obj: data) { internalArray.append(Fleece.toCBLObject(obj)); }
```

Comment:
Can you add @NonNull to data param as well?

Reply:
Yes.  Definitely